ckanext/aberdeen/templates/package/search.html

```diff
@@ -0,0 +1,24 @@
+{% ckan_extends %}                                                                                                                            
+                                                                                                                                              
+{% block package_search_results_api %}                                                                                                        
+  <section class="module">                                                                                                                    
+    <div class="module-content">                                                                                                              
+      {% block package_search_results_api_inner %}                                                                                            
+      <small>                                                                                                                                 
+        {% set api_link = h.link_to(_('API'), 'api/action/status_show') %}                                                                    
```

Comment:
Have you tested this? Should not this be
```
{% set api_link = h.link_to(_('API'), '/api/3/action/status_show') %}   
```
Note the starting `/`

Reply:
Its running with or without the  @`/` locally

Reply:
And the result for `api/3/action/status_show` and `api/action/status_show` is same
Unless it is necessary to add to the link?

Reply:
Let's add starting `/` and 3

Reply:
Pushed https://github.com/datopian/ckanext-aberdeen/pull/6/commits/5f6d509f533cf958873b47c4d6f6255df2db90c3